src/apps/explorer/components/SummaryCardsWidget/summaryGraphResp.json

```diff
@@ -0,0 +1,7 @@
+{
+
+  "batchInfo": { "lastBatchDate":1649881035, "batchId": "0x0003723b9eb1598e12d15c69206bf13c971be0310fa5744cf9601ed79f89e29c" },
+  "dailyTransactions": {"now": 612, "before": 912},
+  "totalTokens": 193,
+  "dailyFees": {"now": 55225.61205047748511254485049406426, "before": 40361.20651840192742698089787142249}
```

Comment:
One more thing about the shape of the data.

Usually it's not a good idea to work with floats in JS and also if working with big numbers JS `number` type won't be able to handle it.

Having said that, it might not be a problem for this use case as:
1. We are simply displaying data without any further calculations, so loss of precision won't be really a problem
2. I expect fees to not be large enough to go over JS `number` limit
3. It mostly depends on how the graph response will look like, which might not be yet known

So, keep that in mind for the future, but does not need necessarily to change anything at this point